# frame_type: 0x37(55)
# speed는 매뉴얼 22pg "jog 시작" 참고 -> 상한: 35000, 하한: 1 
#jog -> 0: -방향, 1: +방향
int32 speed
int32 jog